README.md

```diff
@@ -154,7 +154,7 @@ To run the code generation script, do
     mkdir ../Tmp
     mkdir ../Tmp/data
     mkdir ../Tmp/src
```

Comment:
```suggestion
```

This is also handled by the generator script:
https://github.com/AIDASoft/podio/blob/34ca51e1fc0bcbbe9d594fa66edb8acf8484454f/python/podio_class_generator.py#L533-L536